src/routes/(admin)/questionbank/+page.svelte

```
@@ -1,35 +1,107 @@
+		url.searchParams.set('page', '1');
+
+		goto(url.toString(), { replaceState: false });
+	}
+
+	// Create columns (reactive)
+	const columns = $derived(
+		createQuestionColumns(sortBy, sortOrder, handleSort, currentPage, pageSize)
+	);
+
+	// Check if filters are applied
+	const hasActiveFilters = $derived(() => {
```

ESLint

[error] 'hasActiveFilters' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)